.eslintrc.js

```diff
@@ -6,10 +6,14 @@
  */
 
 module.exports = {
-  extends: 'airbnb-base',
+  extends: [
+    'airbnb-base',
+    "eslint:recommended",
+    "plugin:react/recommended"
+  ],
```

Comment:
I suggest going with the full suite of `airbnb eslint configuration` packages. It extends the linting rules to React, JSX, (ES6+) import/export syntax & others. You can refer to this for more details (https://www.npmjs.com/package/eslint-config-airbnb)